packages/marko/src/runtime/components/init-components-browser.js

```diff
@@ -12,6 +12,10 @@ var addComponentRootToKeyedElements =
   componentsUtil.___addComponentRootToKeyedElements;
 var ComponentDef = require("./ComponentDef");
 var registry = require("./registry");
+var {
+  addPendingDef,
+  hasPendingDef
+} = require("../components/registry-browser");
```

Comment:
Since we do not use babel for the client side runtime, we cannot use object destructuring here.